# message format for raw sensor data
float32[] f0
float32[] f1
float32[] f2
